quarkus/server/build.gradle.kts

```diff
@@ -51,6 +51,10 @@ dependencies {
     runtimeOnly(project(":polaris-eclipselink"))
   }
 
+  if (project.hasProperty("relationalJdbcDeps")) {
+    runtimeOnly(project(":polaris-relational-jdbc"))
```

Comment:
I'm opposed to conditional dependencies.

We should either solidly include it, or not include at all and rely on downstream projects to bundle Postgres Persistence in their builds.

I personally think we should _should_ include this dependency unconditionally.

Reply:
sounds fair, just wanted to have same precedence what we have for eclipse-link, I am fine removing it as well.